roles/identity-management/manage-rh-sso-identities/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Process all tasks
+  include_tasks: process_all_tasks.yml
+  loop: "{{ identities.users }}"
+  loop_control:
+    loop_var: user_data
+  when:
+    - identities.targets is undefined or 'rh-sso' in identities.targets
```

Comment:
Great to see that the `targets` is supported. Make sure this option (`rh-sso`) is documented in the README one-level up.